lib/models/room.js

```diff
@@ -1315,3 +1319,21 @@ module.exports = Room;
  *   }
  * });
  */
+
+/**
+ * Fires when a local-echo of a transmitted event is replaced by its remote-echo.
+ *
+ * <p>When an event is first transmitted, then (assuming the /send completes
```

Comment:
empirically not, and we don't seem to have been closing them in general in the js-sdk doc-comments.